### PR TITLE
W22 5pm 3 ig add images to play page

### LINKED ISSUE #7: Update Play Page User Interface
Update User Interface for Play Page

# Acceptance Criteria:
- [x] Add images to spots where needed, images can be found under frontend/src/assets or you can find ones online but make sure they are free for public use or educational use
- [x] Insert Cash.png
- [x] Insert cowHead.png
- [x] Insert Health.png
- [x] Insert PlayPageBackground.png
- [x] Rearrange UI to make it more intuitive and easy to use but keep all of the same information there

### PR DESCRIPTION
**Pull request by isaiahgarcia**
# Overview

This PR updates the play page UI for Happy Cows. There are 4 inserted images: Cash.png, cowHead.png, Health.png, and PlayPageBackgeround.png. In addition, the "Your Farm Stats" table is moved to the leftside of the Card Group to make the UI experience better. There is also a health bar created for the Cow Health to visually see the percentage. 

# Issues Addressed
#7 

# Details
Screen Shot 2022-03-11 at 4 17 54 AM

### CODE REVIEW

File: frontend/src/main/components/Commons/CommonsOverview.js
Comment (tekaireb) [inline frontend/src/main/components/Commons/CommonsOverview.js]: I think it might be preferable to separate styling from the markup. Perhaps you could consider removing the `<strong>` tag and using a CSS class instead?

File: frontend/src/main/components/Commons/FarmStats.js
Comment (tekaireb) [inline frontend/src/main/components/Commons/FarmStats.js]: You could also reuse the previously suggested CSS class here.

File: frontend/src/main/components/Commons/FarmStats.js
Comment (tekaireb) [inline frontend/src/main/components/Commons/FarmStats.js]: Perhaps you could specify a CSS class here (e.g., `.icon`) to specify the image width and height.

File: frontend/src/main/components/Commons/FarmStats.js
Comment (tekaireb) [inline frontend/src/main/components/Commons/FarmStats.js]: If the suggested CSS class is used here as well, then in the future one could change the width and height values in the CSS file, and the value would change across all icons.

File: frontend/src/main/components/Commons/ManageCows.js
Comment (tekaireb) [inline frontend/src/main/components/Commons/ManageCows.js]: You could also use the same styling class here.

File: frontend/src/main/components/Commons/Profits.js
Comment (tekaireb) [inline frontend/src/main/components/Commons/Profits.js]: The styling class could be used here as well (that way, you can edit the styling in one place, and it will be reflected everywhere).

**Review by tekaireb — CHANGES_REQUESTED** [review]
Nice work!

I suggested a couple changes that would make it easier to change the styling across all components. If you factor the text size, weight, font, etc. into a CSS class, you can reuse that class across similar components. If one wishes to change a value in the future, it will be reflected in every element that makes use of the class automatically. Just something to consider. Otherwise, awesome job!

File: frontend/src/main/components/Commons/FarmStats.js
Comment (pconrad) [reply frontend/src/main/components/Commons/FarmStats.js]: Tyler: at first I was agreeing with you, and even went so far as to look up "what is the bootstrap css class that would apply here".

To my surprise, I found that the way Isaiah wrote the code here actually conforms to what Bootstrap recommends in their own CSS documentation linked here: https://getbootstrap.com/docs/3.4/css/#bold

So, I'm thinking: maybe it's ok to leave it?

I do see your point though, that if we made a CSS class called `.commonsHeaders`, that it would make it easier to style these headers in a single place.  I'll leave it up to the team to decide which way to go.

File: frontend/src/main/components/Commons/FarmStats.js
Comment (pconrad) [reply frontend/src/main/components/Commons/FarmStats.js]: Yes, this is a good point.

File: frontend/src/main/components/Commons/FarmStats.js
Comment (tekaireb) [reply frontend/src/main/components/Commons/FarmStats.js]: Hi @pconrad, that makes sense!

In my review, I assumed that future changes to the styling of all headers was likely (say, increasing font size or adding some other text decoration), so my intuition was to create a CSS class that could be further augmented in a single location.

However, this was just my own intuition; I defer to you and the Bootstrap standard.

I would be happy to approve this as-is when my review is re-requested. Thank you for weighing in!

Comment (isaiahgarcia) [reply frontend/src/main/components/Commons/FarmStats.js]: Hi @pconrad !

The decision was pretty much left up to me after getting feedback from the team. I decided to implement some classes labeled `.icon` and `h5` in index.css.

`.icon` is a class to specify image features. There is only `width` and `height`, which are specified to 'auto' . 

`h5` manipulates the h5 tag by simply using the `font-weight` parameter. I understand that doing this affects ANY text that is specified with an h5 tag, but I think all headers should be bolded anyways. Also, since we are using `Card.Header a="h5`, it makes more sense to abstract the bolding feature into the h5 class itself. 

Thank you for all the feedback, though! It has helped me brush up some knowledge on CSS classes.

Comment (isaiahgarcia) [reply frontend/src/main/components/Commons/FarmStats.js]: @tekaireb ^^ This message also applies to what you suggested. Thanks Tyler!

**Review by tekaireb — DISMISSED** [review]
Really nice work! Looks like all comments have been addressed. Approved!

**Review by pconrad — APPROVED** [review]
Code LGTM---I'll test on Heroku / Storybook and if it works, I'll merge it.

**Pull request merged**
